pkg/ddevapp/drupal.go

```diff
@@ -456,6 +456,11 @@ func writeDrupal6DdevSettingsFile(settings *DrupalSettings, filePath string) err
 // WriteDrushrc writes out drushrc.php based on passed-in values.
 // This works on Drupal 6 and Drupal 7 or with drush8 and older
 func WriteDrushrc(app *DdevApp, filePath string) error {
+	// Ignore because this is a settings file.
+	if app.DisableSettingsManagement {
+		return nil
+	}
```

Comment:
This isn't incorrect IMO, but I think it would be better to fix this in drupal7PostStartAction and also in drupal6PostStartAction. It seems to me like those were both somehow neglected on this. Use drupal8PostStartAction as example.

Congrats on your first golang PR! Please make sure to test it manually.

Reply:
Thanks I'll take a look at that now!